alembic/env.py

```diff
@@ -0,0 +1,83 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from src.city.models import *
+from src.temperature.models import *
```

Comment:
Using wildcard imports (e.g., `from src.city.models import *`) is not recommended as it can lead to namespace pollution and make the code harder to understand. It's better to import only the specific models you need.